lib/gui/app.js

```diff
@@ -173,24 +173,26 @@ app.run(() => {
   store.subscribe(() => {
     const currentFlashState = flashState.getFlashState()
 
+    if (!flashState.isFlashing()) {
```

Comment:
Should we get the `currentFlashState` *after* this check, just to make it more natural?